### PR TITLE
doc(zjow): polish ding model common/template note

### PR DESCRIPTION
**Pull request by zjowowen**

### CODE REVIEW

File: ding/model/common/head.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 class DiscreteHead(nn.Module):
     """
     Overview:
-        The ``DiscreteHead`` used to output discrete actions logit or Q-value logit, which is often used in DQN \
-        and policy head in actor-critic algorithms for discrete action space.
+        The ``DiscreteHead`` is used to generate discrete actions logit or Q-value logit, \
+        which is often used in q-learning algorithmns or actor-critic algorithms for discrete action space.
```
Comment (PaParaZz1) [inline ding/model/common/head.py:18]: typo

File: ding/model/common/head.py [hunk 2]
```diff
@@ -478,7 +476,8 @@ def forward(self, x: torch.Tensor) -> Dict:
 class QuantileHead(nn.Module):
     """
     Overview:
-        The ``QuantileHead`` used to output action quantiles, which is used in IQN.
+        The ``QuantileHead`` is used to output action quantiles.
```
Comment (PaParaZz1) [inline ding/model/common/head.py:479]: add some detailed description about the difference betweene QRDQN/IQN/FQF's head

File: ding/model/common/head.py [hunk 3]
```diff
@@ -1173,7 +1183,8 @@ def forward(self, x: torch.Tensor) -> Dict:
 class PopArtVHead(nn.Module):
     """
     Overview:
-        The ``PopArtVHead`` used to output adaptive normalized state value, which is used in PPO/IMPALA.
+        The ``PopArtVHead`` is used to generate adaptive normalized state value.
```
Comment (PaParaZz1) [inline ding/model/common/head.py:1186]: add paper link

File: ding/model/template/bc.py [hunk 4]
```diff
@@ -188,13 +188,20 @@ def forward(self, inputs: Union[torch.Tensor, Dict[str, torch.Tensor]]) -> Dict:
             - inputs (:obj:`torch.Tensor`): Observation data, defaults to tensor.
         Returns:
             - output (:obj:`Dict`): Output dict data, including different key-values among distinct action_space.
-
+        ReturnsKeys:
+            - action (:obj:`torch.Tensor`): Continuous action output of actor network, \
```
Comment (PaParaZz1) [inline ding/model/template/bc.py:192]: why both continuous here

File: ding/model/template/bcq.py [hunk 5]
```diff
@@ -110,16 +141,63 @@ def compute_critic(self, inputs: Dict[str, torch.Tensor]) -> Dict[str, torch.Ten
...
 
     def compute_vae(self, inputs: Dict[str, torch.Tensor]) -> Dict[str, torch.Tensor]:
+        """
+        Overview:
+            Use vae network to compute action.
+        Arguments:
+            - inputs (:obj:`Dict`): Input dict data, including obs and action tensor.
+        Returns:
+            - outputs (:obj:`Dict`): Dict containing keywords ``recons_action`` (:obj:`torch.Tensor`), \
+            ``prediction_residual`` (:obj:`torch.Tensor`), ``input`` (:obj:`torch.Tensor`), \
```
Comment (PaParaZz1) [inline ding/model/template/bcq.py:173]: there needs some proper indent here

File: ding/model/template/maqac.py [hunk 6]
```diff
@@ -91,7 +91,7 @@ def __init__(
             )
 
     def forward(self, inputs: Union[torch.Tensor, Dict], mode: str) -> Dict:
-        r"""
+        """
```
Comment (PaParaZz1) [inline ding/model/template/maqac.py:94]: You need to polish `maqac` with comparsion to `qac`

File: ding/model/template/ppg.py [hunk 7]
```diff
@@ -43,31 +72,68 @@ def __init__(
...
+        """
         assert mode in self.mode, "not support forward mode: {}/{}".format(mode, self.mode)
         return getattr(self, mode)(inputs)
 
     def compute_actor(self, x: torch.Tensor) -> Dict:
         """
+        Overview:
+            Use actor to compute action logits.
+        Arguments:
+            - x (:obj:`torch.Tensor`): The input data.
```
Comment (PaParaZz1) [inline ding/model/template/ppg.py:92]: don't use such simple description, you should show the meaning of tensor data or the common key-value in dict data

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: ding/model/common/head.py (hunk 3 above)
Comment (zjowowen) [reply ding/model/common/head.py]: fixed

File: ding/model/template/bc.py (hunk 4 above)
Comment (zjowowen) [reply ding/model/template/bc.py]: fixed

File: ding/model/template/bcq.py (hunk 5 above)
Comment (zjowowen) [reply ding/model/template/bcq.py]: fixed

File: ding/model/template/maqac.py (hunk 6 above)
Comment (zjowowen) [reply ding/model/template/maqac.py:97]: fixed

File: ding/model/template/ppg.py (hunk 7 above)
Comment (zjowowen) [reply ding/model/template/ppg.py]: fixed

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**